pkg/sql/colflow/main_test.go

```diff
@@ -80,6 +81,7 @@ func TestMain(m *testing.M) {
 				colexecerror.InternalError(err)
 			}
 		}
+		serverutils.TestingGlobalDRPCOption(base.TestDRPCEnabledRandomly)
```

Comment:
nit: we should do this outside of the anonymous function, right after `InitTestClusterFactory` call.